pkg/compact/compact.go

```diff
@@ -665,7 +665,7 @@ type RetryError struct {
 	err error
 }
 
-func retry(err error) error {
+func Retry(err error) error {
```

Comment:
Does this need to be exported method? If yes add comment, but I would keep it internal if possible

Reply:
As this function is used in another package scope, I need to change it as public. I will add comment for this method.